vktrace/vktrace_replay/vkreplay_vkreplay.cpp

```diff
@@ -2313,6 +2317,11 @@ void vkReplay::manually_replay_vkFreeMemory(packet_vkFreeMemory *pPacket) {
         return;
     }
 
+    if (m_objMapper.m_devicememorys.find(pPacket->memory) == m_objMapper.m_devicememorys.end()) {
+        vktrace_LogError("Skipping vkFreeMemory() due to invalid remapped VkDeviceMemory.");
```

Comment:
Should this be vktrace_LogWarning? It's legal to allocate memory, never use it, and then free the memory, in which case we could generate a warning, not an error.

Reply:
Yes, vktrace_LogWarning should be used here instead of vktrace_LogError.  Will fix it in next commit.